manim/default.cfg

```diff
@@ -147,6 +147,14 @@ streaming_console_banner = Manim is now running in streaming mode.
     >>> c = Circle()
     >>> manim.play(ShowCreation(c))
 
+[custom_folders]  # Syntax : custom_root folder e.g. *Videos* or *Documents/manim* and then the subfolders for the specific scene elements
+custom_root = Videos
+video_dir = temp_files
```

Comment:
Question/nitpick: Why not changing this to something more explicit that will avoid any confusion? Something like `temp_files_video` or something.
I understand that this is meant to be changed, but still.

Reply:
`temp_video_files` would make more sense here but yeah i agree

Reply:
Of course, thank you Captain